src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -657,7 +657,7 @@ private void CompileSynthesizedMethods(PrivateImplementationDetails privateImplC
             Debug.Assert(_moduleBeingBuiltOpt != null);
 
             var compilationState = new TypeCompilationState(null, _compilation, _moduleBeingBuiltOpt);
-            foreach (Cci.IMethodDefinition definition in privateImplClass.GetMethods(new EmitContext(_moduleBeingBuiltOpt, null, diagnostics.DiagnosticBag, metadataOnly: false, includePrivateMembers: true)))
+            foreach (Cci.IMethodDefinition definition in privateImplClass.GetMethods(new EmitContext(_moduleBeingBuiltOpt, null, null, diagnostics.DiagnosticBag, metadataOnly: false, includePrivateMembers: true)))
```

Comment:
nit: it would be nice to label these null literals the same way we label the bool literals

Reply:
Agree. I wanted to make sure we all agreed that `EmitContext` was the right way to flow data here (put acomment in the PR to get clarity on that). If we all agree I'll go back and clean this up. Didn't want to do the work if we were going to decide to thread it through separately cause then I'd have to also undo it.